Framework/SwiftOSC/Communication/OSCServer.swift

```diff
@@ -1,5 +1,10 @@
 import Foundation
 
+
+
+
+
+/// Delegate for OSCServer. In most cases, you only need to implement didReceive(message).
 public protocol OSCServerDelegate: class {
     func didReceive(_ data: Data)
     func didReceive(_ bundle: OSCBundle)
```

Comment:
Receiving bundles. I can't remember if they are delivered before or after the timetag time.

Reply:
Here's from the spec

> When a received OSC Packet contains only a single OSC Message, the OSC Server should invoke the correponding OSC Methods immediately, i.e., as soon as possible after receipt of the packet. Otherwise a received OSC Packet contains an OSC Bundle, in which case the OSC Bundle's OSC Time Tag determines when the OSC Bundle's OSC Messages' corresponding OSC Methods should be invoked. If the time represented by the OSC Time Tag is before or equal to the current time, the OSC Server should invoke the methods immediately

So it sounds like you can schedule a bundle to happen in the future. But if the timetag is less than or equal to the current time, it's executed immediately.

I'll update a few parts with this info.